README.md

```diff
@@ -102,7 +102,17 @@ nginx is configured to use default port 80.
 
 ### Testing
 
-TBD
+Install pytest for our premade tests.
+
+```bash
+pipenv install pytest
+```
+Make sure the DATABASE_URL is correct, and it can run normally in flask. Run with:
```

Comment:
Nit: Also probably a good idea to set the `DATABASE_URL` to point to a `test` database:
```bash
export DATABASE_URL=postgres://postgres:postgres@127.0.0.1:5432/test
```

Reply:
Ill add a comment to that extent